### PR TITLE
Add initial round of new styling and home page layout

### PR DESCRIPTION
**Pull request by eshaben**
This PR updates the general styling throughout the docs as well as the home page layout and styling. There were some issues with the animation, so I'll add that in a separate PR

Goes with polkadot-docs PR: https://github.com/polkadot-developers/polkadot-docs/pull/122

### CODE REVIEW

**Review by albertov19 — CHANGES_REQUESTED** [review]
Looks good!!! 🔥 

Some minor comments. I think we need something different at the top of the main page as the navigation pill goes over the text: 
image

We should add "Development Pathways" to the menu that expand for "Develop"

image


When I go to that page, the left hand menu breaks:

image

I feel for whatever reason that the text margins in the center is very tight:

image

File: material-overrides/assets/stylesheets/home.css [hunk 1]
```diff
@@ -62,59 +154,151 @@
...
   }
 
-  .feature {
-    width: 50%;
+  .hero-content h1 {
+    font-size: 4rem;
+    line-height: 1;
+    margin-top: 1rem;
+    margin-bottom: 2rem;
+  };
```
Comment (nhussein11) [inline material-overrides/assets/stylesheets/home.css:249]:
```suggestion
  .hero-content h1 {
    font-size: 4rem;
    line-height: 1;
    margin-top: 1rem;
    margin-bottom: 2rem;
  }
```

File: material-overrides/assets/stylesheets/home.css [hunk 2]
```diff
@@ -62,59 +154,151 @@
...
+  }
+
+  .md-typeset a.feature {
+    min-width: 60%;
+    max-width: 75%;
+  }
+
+  .md-typeset a.feature h3 {
+    font-size: 1.5rem;
+  };
```
Comment (nhussein11) [inline material-overrides/assets/stylesheets/home.css:272]:
```suggestion
  .md-typeset a.feature h3 {
    font-size: 1.5rem;
  }
```

**Comment by nhussein11** [comment]
looking goooood! 👏

**Comment by eshaben** [comment]
@albertov19 

> Some minor comments. I think we need something different at the top of the main page as the navigation pill goes over the text:

Yes, that is how they have it on polkadot.com, so I just followed suit. I can ask iggy if he wants something else

> We should add "Development Pathways" to the menu that expand for "Develop"

This is auto-generated based on the current IA. I can probably add Development Pathways as it's own section with just an index page. No other pages would show up on the left nav once clicked on that page though. So, we could do like a second home page if we find that we want it styled a bit more than the usual index pages, but of course, that would take more time and require a design. Let me know what you think or what you'd like to see when you click on Development Pathways

> When I go to that page, the left hand menu breaks:

There is no content for this page yet, and the left nav is out of sorts since we're midway through porting the content over. I can try to clean this up today so it doesn't look like everything is breaking.

> I feel for whatever reason that the text margins in the center is very tight:

Weird that it looks like this for you! What size screen was this screenshot taken from? It looks like it might be like a half screen judging by the top nav. I can apply some screen size-specific styling here

**Review by albertov19 — APPROVED** [review]

**Pull request merged**